### ISSUE TITLE
Automate CI Build for Existing Targets

### ISSUE THREAD

**Issue by aspickard** [dasher-project/redash#71]
{NOTE} CI currently running for Web Target on Travis-CI. (iOS in progress..)
No static analysis is setup or running.

**Comment by gavinhenderson** [comment]
Just wanting to start around automation, I know we might get round to it for a while.

Are we set on continuing to use Travis CI for our automation? 

I have been using Github actions a lot for projects lately and its really good. There is a huge marketplace of prebuilt actions you can pull from, and there is bound to be stuff for iOS build and electron builds that we can use.

My vote would be to migrate to Actions

**Comment by aspickard** [comment]
+1 for utilizing actions, but that does lock us in to GitHub

**Comment by gavinhenderson** [comment]
Yeah that's a good point but I think we are already pretty locked into GitHub anyway with the amount of stuff we have on it so we might as well double down

**Comment by jcope** [comment]
I second moving it over to GitHub actions- currently it is set up on my Travis CI account, but needs updating to reflect the new repo layout. I having used the actions before, but it seems similar and should support our environment.

**Comment by gavinhenderson** [comment]
- [ ] Signing stuff
- [ ] Add a simple link to website to download binaries (for folk who are afraid of github)
